sentences/cs/_common.yaml

```diff
@@ -7,6 +7,58 @@ responses:
     no_device_class: "{{ area }} neobsahuje {{ device_class }}"
     no_entity: "Žádné zařízení ani entita se nejmenuje {{ entity }}"
     handle_error: "Při zpracování došlo k neočekávané chybě."
-lists: {}
-expansion_rules: {}
-skip_words: []
+lists:
+  color:
+    values:
+      - in: "bíl(á|é|ý|ou)"
+        out: "white"
+      - in: "čern(á|é|ý|ou)"
+        out: "black"
+      - in: "červen(á|é|ý|ou)"
+        out: "red"
+      - in: "oranžov(á|é|ý|ou)"
+        out: "orange"
+      - in: "žlut(á|é|ý|ou)"
+        out: "yellow"
+      - in: "zelen(á|é|ý|ou)"
+        out: "green"
+      - in: "modr(á|é|ý|ou)"
+        out: "blue"
+      - in: "fialov(á|é|ý|ou)"
+        out: "purple"
+      - in: "hněd(á|é|ý|ou)"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "Celsius"
```

Comment:
Shouldn't this be rather Celsia, since we sat "18 stupňů Celsia"?

Reply:
Sorry, don't know how to update the patch in pull request.

Yes "Celsia" is better

Reply:
@zenhas I think you should create the PR from branch in your fork, not just from commits https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

Reply:
@TomaszPilch  Will do in the future. 
Thought that when i make fork and work on is enought. I created PR from my forked master. Anyway, next time I'll make branch and request merge from that.
Thanks.